x-pack/legacy/plugins/infra/public/pages/logs/analysis/setup/steps/step_text.tsx

```diff
@@ -0,0 +1,20 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License;
+ * you may not use this file except in compliance with the Elastic License.
+ */
+
+import { EuiText } from '@elastic/eui';
+import euiStyled from '../../../../../../../../common/eui_styled_components';
+
+export const StepText = euiStyled(EuiText).attrs({ size: 's' })`
+  & .euiButtonEmpty {
```

Comment:
Could we achieve the same using just the [`EuiLink`](https://elastic.github.io/eui/#/navigation/link) component?

Reply:
I've changed this over. The downside, for all it saves some custom styling, is it's not semantically correct anymore 😬

Reply:
Isn't it rendered as button element when an `onClick` prop is passed? At least that's what the EUI docs suggest.

Reply:
Ah, so it is, TIL 😄